recipes/crypten/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "crypten" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: f9d76bc0ba1468cb77261b8d5486ee875266363310353a18aba948429aa24fda
+  - url: https://raw.githubusercontent.com/facebookresearch/CrypTen/main/requirements.txt
```

Comment:
Why do you need this file? It repeats the information in the `meta.yaml` file

Reply:
Yes, I will make a PR once this recipe is ready, and then make the change appropriately.

Reply:
this recipe is nearly ready once you remove the `requirements.txt` source. You can then remove one level of nesting in your url list.

Reply:
Yes, but since I am not sure when the other PR would get merged, I was hoping to have the recipe available. I could then make for a stronger case for the PR.

Reply:
I have added the PR: https://github.com/facebookresearch/CrypTen/pull/350

Reply:
You don't need to make the PR. that file is not necessary to be distributed. it is distributed as metadata along with the wheel or the conda package. Simply remove it from the recipe. You've copied the contents in the meta already.

Reply:
Ahh that is too bad. Yeah. this seems OK. however you aren't taking a specific tag or commit. I suspect this will be problematic in the future. You should for yourself include a cross refernece to the issue / PR you raised upstream. But this looks good now.